packages/app/universal-ts-utils/src/array/unique.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * A method to remove duplicates from an array of primitive values.
+ * Elements are compared by reference using a Set.
```

Comment:
is that true? isn't it done using native equality check, which for objects relies on a reference check, but for primitives does not?

Reply:
It's actually using SameValueZero algorithm. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#value_equality